package.json

```diff
@@ -30,6 +31,7 @@
     "node": ">=4"
   },
   "dependencies": {
+    "@types/jquery": "^2.0.39",
```

Comment:
Unsure about adding this as a @type dependency. Does it affect `npm` or any other package management system?